flare/agent_zoo/simple_rl_agents.py

```diff
@@ -62,3 +64,35 @@ def _cts_predict(self, observations, states):
         actions, next_states = self.predict(
             'RL', inputs=dict(sensor=observations), states=dict(state=states))
         return [actions.values()[0][0]], [next_states.values()[0][0]]
+
+
+class ActionNoiseAgent(SimpleRLAgent):
+    """
+    This class extends `SimpleRLAgent` by applying action noise after 
+    prediction. It can be used to algorithms with deterministic policies, e.g.,
+    `DDPG`.
+    """
+
+    def __init__(self, env, num_games, action_noise):
+        super(ActionNoiseAgent, self).__init__(env, num_games)
+        self.action_noise = action_noise
+
+    def _cts_store_data(self, observations, actions, states, rewards):
+        assert len(observations) == 1 and len(actions) == 1
+        self._store_data(
+            'RL',
+            dict(
+                sensor=observations[0], action=actions[0], reward=rewards))
+
+    def _cts_predict(self, observations, states):
+        ## each action is already 2D
```

Comment:
Comment confusing. Do you want to say that "the return actions is already a 2D tensor? "